docs/UserGuide.md

```diff
@@ -417,11 +447,34 @@ Add multiple residents to an event based on the information given(name or room n
 
 Format: `include INDEX r/RESIDENTS`
 
-* Residents can be given in the form of name or room, but all has to be all rooms or all names
-* The resident's name/room inputted has to exist in the address book under the `Resident` Tab to be added to an `Event`  
-* When adding multiple names/rooms, each resident's name/room is separated by a comma
-* The resident's name/room inputted is case-insensitive
-* If one or more of the given name/room is invalid, an error message is outputted and none of the name/room is added to the event
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+
+* Residents can be given in the form of `names/rooms`, but all has to be all `rooms` or all `names`
+* The residents' `names/rooms` inputted have to exist in the address book under the `Resident` Tab to be added to an `Event`
+* When adding multiple `names/rooms`, each resident's `name/room` is separated by a comma
+* The resident's `name/room` inputted is case-insensitive
+
+</div>
+
+Here's a step by step guide:<br>
+1. Navigate to the `Event` Tab
+   ![Step1](images/logic/commands/includecommand/step1.png)
+   
+
+2. View the event to add residents to, either through the `view` command or by clicking on the desired event
```

Comment:
Since the `view` command is not needed to be run for this command, this might be misleading. Maybe remove this step or mention that it's upto the user's discretion if they want to first see more information

Reply:
Alright